### PR TITLE
Fixed wrong copy/paste

### PR DESCRIPTION
**Pull request by worktycho**
Fixes CID 110962

### CODE REVIEW

File: src/OverridesSettingsRepository.cpp [hunk 1]
```diff
@@ -260,7 +260,7 @@ bool cOverridesSettingsRepository::DeleteValue(const AString & a_KeyName, const
 	}
 	else
 	{
-		return m_Overrides->DeleteValue(a_KeyName, a_ValueName);
+		return m-Main->DeleteValue(a_KeyName, a_ValueName);
```
Comment (NiLSPACE) [inline src/OverridesSettingsRepository.cpp]: I assume it's `a_Main` not `a-Main`?

Comment (worktycho) [inline src/OverridesSettingsRepository.cpp]: Yep, fixing.

**Comment by worktycho** [comment]
I think I need to give up using the web editor to make changes, I always make a smaller error that needs a new commit to be squashed in.

**Comment by worktycho** [comment]
Fixed.

**Pull request merged**